### PR TITLE
Added table aws_ec2_network_load_balancer_metric_net_flow_count and aws_ec2_application_load_balancer_metric_request_count. Closes #521, Closes #520

### LINKED ISSUE #520: Add table aws_ec2_application_load_balancer_metric_data
**References**
https://docs.aws.amazon.com/elasticloadbalancing/latest/application/load-balancer-cloudwatch-metrics.html

https://docs.aws.amazon.com/elasticloadbalancing/latest/application/load-balancer-cloudwatch-metrics.html#metric-statistics

https://www.cloudconformity.com/knowledge-base/aws/ELBv2/idle-load-balancer.html
**Use case:**
By default, an AWS ALB resource is considered "idle" when it meets the following criteria:

The sum of the requests made to the load balancer in the past 7 days is less than 100. The CloudWatch metric used to detect idle Application Load Balancers is RequestCount (Sum). This metric records the number of HTTP/HTTPS requests processed over a predefined time frame. It includes only the requests with a response generated by a target of the load balancer.

### LINKED ISSUE #521: Add table aws_ec2_network_load_balancer_metric_data
**References**
https://docs.aws.amazon.com/elasticloadbalancing/latest/network/load-balancer-cloudwatch-metrics.html
**Use case:**
An AWS NLB resource is considered "idle" when it meets the following criteria:
The sum of the TCP connections initiated in the past 7 days is less than 100. The CloudWatch metric used to detect idle Network Load Balancers is NewFlowCount (Sum). This metric records the number of TCP connections (flows) established from clients to targets within a predefined time frame.

### PR DESCRIPTION
**Pull request by khushboo9024**
# Integration test logs
<details>
  <summary>Logs</summary>
  
```
NA
```
</details>

# Example query results
<details>
  <summary>Results</summary>
  
```
```sql
select
  name,
  metric_name,
  namespace,
  maximum,
  minimum,
  sample_count,
  timestamp
from
  aws_ec2_network_load_balancer_metric_net_flow_count
order by
  name,
  timestamp;
 +------------------------------+--------------+----------------+---------+---------+--------------+---------------------+
| name                         | metric_name  | namespace      | maximum | minimum | sample_count | timestamp           |
+------------------------------+--------------+----------------+---------+---------+--------------+---------------------+
| net/test566/d125965c5985e971 | NewFlowCount | AWS/NetworkELB | 0       | 0       | 1            | 2021-08-09 12:03:00 |
| net/test566/d125965c5985e971 | NewFlowCount | AWS/NetworkELB | 0       | 0       | 5            | 2021-08-09 12:08:00 |
| net/test566/d125965c5985e971 | NewFlowCount | AWS/NetworkELB | 0       | 0       | 5            | 2021-08-09 12:13:00 |
| net/test566/d125965c5985e971 | NewFlowCount | AWS/NetworkELB | 0       | 0       | 5            | 2021-08-09 12:18:00 |
| net/test566/d125965c5985e971 | NewFlowCount | AWS/NetworkELB | 0       | 0       | 5            | 2021-08-09 12:23:00 |
| net/test566/d125965c5985e971 | NewFlowCount | AWS/NetworkELB | 0       | 0       | 5            | 2021-08-09 12:28:00 |
| net/test566/d125965c5985e971 | NewFlowCount | AWS/NetworkELB | 0       | 0       | 5            | 2021-08-09 12:33:00 |
| net/test566/d125965c5985e971 | NewFlowCount | AWS/NetworkELB | 0       | 0       | 5            | 2021-08-09 12:38:00 |
| net/test566/d125965c5985e971 | NewFlowCount | AWS/NetworkELB | 0       | 0       | 5            | 2021-08-09 12:43:00 |
| net/test566/d125965c5985e971 | NewFlowCount | AWS/NetworkELB | 0       | 0       | 5            | 2021-08-09 12:48:00 |
| net/test566/d125965c5985e971 | NewFlowCount | AWS/NetworkELB | 0       | 0       | 5            | 2021-08-09 12:53:00 |
| net/test566/d125965c5985e971 | NewFlowCount | AWS/NetworkELB | 0       | 0       | 5            | 2021-08-09 12:58:00 |
| net/test566/d125965c5985e971 | NewFlowCount | AWS/NetworkELB | 0       | 0       | 5            | 2021-08-09 13:03:00 |
| net/test566/d125965c5985e971 | NewFlowCount | AWS/NetworkELB | 0       | 0       | 5            | 2021-08-09 13:08:00 |
| net/test566/d125965c5985e971 | NewFlowCount | AWS/NetworkELB | 0       | 0       | 5            | 2021-08-09 13:13:00 |
| net/test566/d125965c5985e971 | NewFlowCount | AWS/NetworkELB | 0       | 0       | 5            | 2021-08-09 13:18:00 |
| net/test566/d125965c5985e971 | NewFlowCount | AWS/NetworkELB | 0       | 0       | 5            | 2021-08-09 13:23:00 |
| net/test566/d125965c5985e971 | NewFlowCount | AWS/NetworkELB | 0       | 0       | 5            | 2021-08-09 13:28:00 |
| net/test566/d125965c5985e971 | NewFlowCount | AWS/NetworkELB | 0       | 0       | 5            | 2021-08-09 13:33:00 |
| net/test566/d125965c5985e971 | NewFlowCount | AWS/NetworkELB | 0       | 0       | 5            | 2021-08-09 13:38:00 |
| net/test566/d125965c5985e971 | NewFlowCount | AWS/NetworkELB | 0       | 0       | 5            | 2021-08-09 13:43:00 |
| net/test566/d125965c5985e971 | NewFlowCount | AWS/NetworkELB | 0       | 0       | 5            | 2021-08-09 13:48:00 |
| net/test566/d125965c5985e971 | NewFlowCount | AWS/NetworkELB | 0       | 0       | 5            | 2021-08-09 13:53:00 |
| net/test566/d125965c5985e971 | NewFlowCount | AWS/NetworkELB | 0       | 0       | 5            | 2021-08-09 13:58:00 |
| net/test566/d125965c5985e971 | NewFlowCount | AWS/NetworkELB | 0       | 0       | 5            | 2021-08-09 14:03:00 |
| net/test566/d125965c5985e971 | NewFlowCount | AWS/NetworkELB | 0       | 0       | 5            | 2021-08-09 14:08:00 |
| net/test566/d125965c5985e971 | NewFlowCount | AWS/NetworkELB | 0       | 0       | 5            | 2021-08-09 14:13:00 |
| net/test566/d125965c5985e971 | NewFlowCount | AWS/NetworkELB | 0       | 0       | 5            | 2021-08-09 14:18:00 |
| net/test566/d125965c5985e971 | NewFlowCount | AWS/NetworkELB | 0       | 0       | 5            | 2021-08-09 14:23:00 |
| net/test566/d125965c5985e971 | NewFlowCount | AWS/NetworkELB | 0       | 0       | 5            | 2021-08-09 14:28:00 |
| net/test566/d125965c5985e971 | NewFlowCount | AWS/NetworkELB | 0       | 0       | 5            | 2021-08-09 14:33:00 |
| net/test566/d125965c5985e971 | NewFlowCount | AWS/NetworkELB | 0       | 0       | 5            | 2021-08-09 14:38:00 |
| net/test566/d125965c5985e971 | NewFlowCount | AWS/NetworkELB | 0       | 0       | 5            | 2021-08-09 14:43:00 |
| net/test566/d125965c5985e971 | NewFlowCount | AWS/NetworkELB | 0       | 0       | 5            | 2021-08-09 14:48:00 |
| net/test566/d125965c5985e971 | NewFlowCount | AWS/NetworkELB | 0       | 0       | 2            | 2021-08-09 14:53:00 |
+------------------------------+--------------+----------------+---------+---------+--------------+---------------------+

```
```
</details>

### CODE REVIEW

File: aws/table_aws_ec2_application_load_balancer_metric_ request_count.go [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+	"github.com/turbot/steampipe-plugin-sdk/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/plugin/transform"
+)
+
+//// TABLE DEFINITION
+func tableAwsEc2ApplicationLoadBalancerMetricRequestCount(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "aws_ec2_application_load_balancer_metric_request_count",
+		Description: "AWS EC2 Application LoadBalancer Metrics - Request Count",
```
Comment (bigdatasourav) [inline aws/table_aws_ec2_application_load_balancer_metric_ request_count.go:17]:
```suggestion
		Description: "AWS EC2 Application Load Balancer Metrics - Request Count",
```

File: aws/table_aws_ec2_network_load_balancer_metric_net_flow_count.go [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+	"github.com/turbot/steampipe-plugin-sdk/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/plugin/transform"
+)
+
+//// TABLE DEFINITION
+func tableAwsEc2NetworkLoadBalancerMetricNetFlowCount(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "aws_ec2_network_load_balancer_metric_net_flow_count",
+		Description: "AWS EC2 Network LoadBalancer Metrics - Net Flow Count",
```
Comment (bigdatasourav) [inline aws/table_aws_ec2_network_load_balancer_metric_net_flow_count.go:17]:
```suggestion
		Description: "AWS EC2 Network Load Balancer Metrics - Net Flow Count",
```

**Review by bigdatasourav — APPROVED** [review]
LGTM

**Review by bigdatasourav — CHANGES_REQUESTED** [review]
@khushboo9024 please add doc files and example queries for this 2 tables

**Review by bigdatasourav — APPROVED** [review]
LGTM

**Review by rajlearner17 — APPROVED** [review]
LGTM

**Pull request merged**